.github/ISSUE_TEMPLATE/feature-request.yaml

```
@@ -0,0 +1,32 @@
+name: Feature request
+description: Add a request for a new feature or a change
+title: 'Feature request'
+body:
+  - type: markdown
+    attributes:
+      value: Please provide a concise set of requirements, so that we can process your request as soon as possible.
+  - type: textarea
+    id: user-story
+    attributes:
+      label: User story
+      description: Here you can phrase your request in a user story format which highlights the added value of the requested feature or change.
+      placeholder: e.g. As a Code PushUp user, I want to <request>, so that <added value>.
+    validations:
+      required: true
+  - type: textarea
+    id: acceptance-criteria
+    attributes:
+      label: Acceptance criteria
+      description: Please list a set of criteria to help us determine whether the scope of the request was fully addressed.
+      placeholder: |
+        For example:
+        - All packages have a comprehensive documentation in place.
+        - Cross references to relevant models or files are added as links.
```

I usually use checkboxes for acceptance criteria:

```suggestion
        - [ ] All packages have a comprehensive documentation in place.
        - [ ] Cross references to relevant models or files are added as links.
```